tests/atest/rules/misc/inconsistent_assignment_in_variables/autodetect/expected_output.txt

```diff
@@ -1 +1 @@
-autodetect${/}test.robot:7:9 [W] 0910 The assignment sign is not consistent inside the variables section. Expected ' =' but got '=' instead
\ No newline at end of file
+autodetect${/}test.robot:7:1 [W] 0910 The assignment sign is not consistent inside the variables section. Expected ' =' but got '=' instead
```

Comment:
The start of the line over the place where the assignment sign is?

Reply:
Yes, because the variables in Variables section start at the beginning of the line.

<img width="621" alt="image" src="https://user-images.githubusercontent.com/7412964/226834465-72d760fa-1f47-44e8-95ad-a9d638407ffa.png">